resources/views/tasks/edit.blade.php

```diff
@@ -868,6 +869,16 @@ class="mr-2 custom-badges pl-2 pr-2 rounded-lg">
           },
           onInput(filter) {
             this.getUsers(filter);
+          },
+          getCommentsData: async () => {
```

Comment:
missing catch error

Reply:
The commentContainer component has the success handler for this method, for else it does nothing.